### PR TITLE
Replace arg `keep_prob` with `p` in mindspore.nn.Dropout

### PR DESCRIPTION
**Pull request by horcham**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan
1.  detection model. train and evaluate on icdar15 dataset: 
     - [x] dbnet-resnet50, f-score: 0.846(85.05% expected). 
3.  recognition model, train and evaluate svtr-tiny on lmdb dataset: 
     - [x] crnn-resnet34(epoch 20/30), Avg acc: 0.849(84.45% expected)
     - [ ] svtr-tiny(epoch 18/30), Avg acc: 0.880(90.23% expected)
## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: mindocr/models/backbones/mindcv_models/layers/drop_path.py [hunk 1]
```diff
@@ -20,10 +18,7 @@ def __init__(
         super().__init__()
         self.keep_prob = 1.0 - drop_prob
         self.scale_by_keep = scale_by_keep
-        if is_ms_version_2():
-            self.dropout = nn.Dropout(p=1 - self.keep_prob)
-        else:
-            self.dropout = nn.Dropout(keep_prob=self.keep_prob)
+        self.dropout = nn.Dropout(p=1 - self.keep_prob)
```
Comment (chengtianwu) [inline mindocr/models/backbones/mindcv_models/layers/drop_path.py:21]: 和19行是不是可以合并？

Comment (horcham) [reply mindocr/models/backbones/mindcv_models/layers/drop_path.py]: `self.keep_prob`在`construct`被使用，已然保留。这一行改为`self.dropout = nn.Dropout(p=drop_prob)`

**Review by tonytonglt — APPROVED** [review]

**Review by chengtianwu — APPROVED** [review]

File: mindocr/models/backbones/mindcv_models/ghostnet.py [hunk 2]
```diff
@@ -227,7 +227,7 @@ def __init__(
                                    padding=0, stride=1, has_bias=True, pad_mode="pad")
         self.act2 = nn.ReLU()
         if self.dropout_rate > 0:
-            self.dropout = nn.Dropout(self.dropout_rate)
+            self.dropout = nn.Dropout(p=self.dropout_rate)
```
Comment (panshaowu) [inline mindocr/models/backbones/mindcv_models/ghostnet.py:230]: 此处不需要变换吗，那为何googlenet中的需要变换。

Comment (horcham) [reply mindocr/models/backbones/mindcv_models/ghostnet.py:230]: 这里的修改是与mindcv仓中`mindnv/mindcv/models/ghostnet.py:256`行对齐
https://github.com/mindspore-lab/mindcv/blob/main/mindcv/models/ghostnet.py
